src/pages/CollectionPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+import { Container, Row, Col, Card } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../../components/HeaderBar/HeaderBar"
+import { getCollections } from "../../api/collections-api";
+import { Link } from "react-router-dom";
+import placeholderImg from '../../asset/placehoderImg.jpg';
+
+const CollectionsPage = () => {
+    const [result, setResult] = useState<Collection[]>([])
+
+    useEffect(() => {
+        getCollections().then((response) => {
+            setResult(response.result)
+        })
+    }, [])
+
+    return (
+        <>
+            <HeaderBar />
+            <Container>
+                <Row>
+                    {!!result.length && result.map((item: Collection, index) => {
```

Comment:
what's !! doing?

Reply:
Without !! React thinks 'result.length' is a number first. So it displays '0' in the browser with 'Loading...'. By adding !! React knows I meant boolean from the get go.

Reply:
https://betterprogramming.pub/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054